docs(tests): added debug instructions for VSCode (#3612)
invertase/react-native-firebase: tests/README.md

---

> ### Debugging (VsCode)

Reviewer: ```suggestion
### Debugging E2E JS Tests (VSCode)
```

1. Add a breakpoint in the JS file where you will need to debug. 

> 2. Select the debug icon under 'NPM Scripts', choosing one of the following...

Reviewer: ```suggestion
2. Select the debug icon under 'NPM Scripts' when selecting one of the following scripts...
```

>   - tests:ios:test:debug (IOS)

Reviewer: ```suggestion
  - tests:ios:test:debug (iOS)
```
